improver/cli/estimate_emos_coefficients.py

```diff
@@ -13,9 +13,14 @@
 @cli.clizefy
 @cli.with_output
 def process(
-    *cubes: cli.inputcube,
+    forecast_directory: cli.inputpath,
+    truth_directory: cli.inputpath,
+    land_sea_mask: cli.inputcube = None,
```

Comment:
Caveat to this.  I'm not a fan of the use of the clize typing mechanism within IMPROVER.  However, this might facilitate maintaining existing CLI functionality and behaviour while also facilitating separating 'load' from the plugin itself.